bundles/org.openhab.binding.monopriceaudio/src/main/java/org/openhab/binding/monopriceaudio/internal/handler/MonopriceAudioHandler.java

```diff
@@ -0,0 +1,723 @@
+
+    private void processZoneUpdate(MonopriceAudioZone zone, MonopriceAudioZoneDTO zoneData, String newZoneData) {
+        // only process the update if something actually changed in this zone since the last time through
+        if (!newZoneData.equals(zoneData.toString())) {
+            // example status string: 1200010000130809100601, matcher pattern from above:
+            // "^(\\d{2})(\\d{2})(\\d{2})(\\d{2})(\\d{2})(\\d{2})(\\d{2})(\\d{2})(\\d{2})(\\d{2})(\\d{2})"
+            Matcher matcher = PATTERN.matcher(newZoneData);
+            if (matcher.find()) {
+                zoneData.setZone(matcher.group(1));
+
+                if (!matcher.group(2).equals(zoneData.getPage())) {
+                    zoneData.setPage(matcher.group(2));
+                    updateChannelState(zone, CHANNEL_TYPE_PAGE, zoneData);
+                }
+
+                if (!matcher.group(3).equals(zoneData.getPower())) {
+                    zoneData.setPower(matcher.group(3));
+                    updateChannelState(zone, CHANNEL_TYPE_POWER, zoneData);
+                }
+
+                if (!matcher.group(4).equals(zoneData.getMute())) {
+                    zoneData.setMute(matcher.group(4));
+                    updateChannelState(zone, CHANNEL_TYPE_MUTE, zoneData);
+                }
+
+                if (!matcher.group(5).equals(zoneData.getDnd())) {
+                    zoneData.setDnd(matcher.group(5));
+                    updateChannelState(zone, CHANNEL_TYPE_DND, zoneData);
+                }
+
+                if (!matcher.group(6).equals(zoneData.getVolume())) {
+                    zoneData.setVolume(matcher.group(6));
+                    updateChannelState(zone, CHANNEL_TYPE_VOLUME, zoneData);
+                }
+
+                if (!matcher.group(7).equals(zoneData.getTreble())) {
+                    zoneData.setTreble(matcher.group(7));
+                    updateChannelState(zone, CHANNEL_TYPE_TREBLE, zoneData);
+                }
+
+                if (!matcher.group(8).equals(zoneData.getBass())) {
+                    zoneData.setBass(matcher.group(8));
+                    updateChannelState(zone, CHANNEL_TYPE_BASS, zoneData);
+                }
+
+                if (!matcher.group(9).equals(zoneData.getBalance())) {
+                    zoneData.setBalance(matcher.group(9));
+                    updateChannelState(zone, CHANNEL_TYPE_BALANCE, zoneData);
+                }
+
+                if (!matcher.group(10).equals(zoneData.getSource())) {
+                    zoneData.setSource(matcher.group(10));
+                    updateChannelState(zone, CHANNEL_TYPE_SOURCE, zoneData);
+                }
+
+                if (!matcher.group(11).equals(zoneData.getKeypad())) {
+                    zoneData.setKeypad(matcher.group(11));
+                    updateChannelState(zone, CHANNEL_TYPE_KEYPAD, zoneData);
+                }
+            } else {
+                logger.warn("Invalid zone update message: {}", newZoneData);
```

Comment:
This is more of an external issue rather than one that a user can address, so just make this log debug instead.
```suggestion
                logger.debug("Invalid zone update message: {}", newZoneData);
```